src/Http/Http.Results/src/HttpResultsHelper.cs

```diff
@@ -34,8 +38,8 @@ internal static partial class HttpResultsHelper
         jsonSerializerOptions ??= ResolveJsonOptions(httpContext).SerializerOptions;
         var jsonTypeInfo = (JsonTypeInfo<TValue>)jsonSerializerOptions.GetTypeInfo(typeof(TValue));
 
-        Type? runtimeType;
-        if (jsonTypeInfo.IsValid(runtimeType = value.GetType()))
+        Type? runtimeType = value.GetType();
+        if (jsonTypeInfo.IsValid(runtimeType))
```

Comment:
Nit: naming
```suggestion
        if (jsonTypeInfo.ShouldUseWith(runtimeType))
```

Reply:
He’s suggesting renaming IsValid to ShouldUseWith

Reply:
🤦🏽‍♀️ In that case, I think that's a great recommendation. Reads much more nicely...